specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Dataset.json

```diff
@@ -573,7 +573,11 @@
         },
         "sheetName": {
           "type": "object",
-          "description": "The sheet of excel file. Type: string (or Expression with resultType string)."
+          "description": "The sheet name of excel file. Type: string (or Expression with resultType string)."
+        },
+        "sheetIndex": {
```

Comment:
This might be considered as a braking change?

Reply:
Hi @MushMal, we received issues regarding this property. We add this property to fix the SDK inconsistency between ADF and Synapse. The breaking changes have been approved.